### PR TITLE
nnue: june 2026

### PR DESCRIPTION
**Pull request by jorgenhanssen**
New datagen and trained net.

Samples: 1,001,083,382
Games: 32,267,819

20% WDL
74 epochs

Train Loss and Val Loss

### CODE REVIEW

**Comment by jorgenhanssen** [comment]
```
--------------------------------------------------
Results of grail vs grail-next (60+0.6, 1t, 256MB, UHO_Lichess_4852_v1.epd):
Elo: 19.15 +/- 17.45, nElo: 35.18 +/- 31.96
LOS: 98.45 %, DrawRatio: 47.58 %, PairsRatio: 1.59
Games: 454, Wins: 147, Losses: 122, Draws: 185, Points: 239.5 (52.75 %)
Ptnml(0-2): [4, 42, 108, 71, 2], WL/DD Ratio: 2.00
LLR: 0.60 (20.3%) (-2.94, 2.94) [0.00, 5.00]
--------------------------------------------------
```
